db/migrate/20151001084315_create_missions.rb

```diff
@@ -0,0 +1,13 @@
+class CreateMissions < ActiveRecord::Migration
+  def change
+    drop_table :tasks
+    create_table :missions do |t|
+      t.string :target
+      t.string :description
+      t.string :status
+      t.timestamps null: false
+      add_reference :missions, :user, index: true
+      add_foreign_key :missions, :users
```

Comment:
These two lines should not be in this method. Actually, if I were you, I'll use `t.refernces :missions, index: true, foreign_key: true` instead.